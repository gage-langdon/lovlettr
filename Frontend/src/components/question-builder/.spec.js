import React from 'react';
import { shallow } from 'enzyme';
//component
import QuestionBuilder from './index';

describe('Question component', () => {
  it('Will only render with all props passed in', () => {
    const wrap = shallow(<QuestionBuilder />);
    expect(wrap).toMatchSnapshot();
  });
});
