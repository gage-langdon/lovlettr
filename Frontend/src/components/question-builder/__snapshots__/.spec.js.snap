// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Question component Will only render with all props passed in 1`] = `
<div
  className="d-flex flex-column"
>
  <form
    onSubmit={[Function]}
  >
    <Input
      className="input-question-field"
      disabled={false}
      placeholder="Will you go on a date with me?"
      type="text"
      value=""
    />
  </form>
  <sub
    className="question-sub-text"
    hidden={false}
  >
    enter a question and then add some answers
  </sub>
  <div
    className="d-flex flex-column"
  >
    <AddAnswer
      answerInputText=""
      answerItemCreate={[Function]}
      answers={Array []}
      disabled={true}
      hidden={true}
      label="+ add answer"
      questionInputText=""
    />
  </div>
</div>
`;
